src/System.Private.CoreLib/shared/System/AppDomain.cs

```diff
@@ -199,7 +199,7 @@ public static bool MonitoringIsEnabled
 
         public static long MonitoringSurvivedProcessMemorySize { get { throw CreateResMonNotAvailException(); } }
```

Comment:
You can also implement this one as `GC.GetGCMemoryInfo().HeapSizeBytes`

Reply:
Cool, was wondering about that, will do.

Reply:
Since there aren't multiple AppDomains, MonitoringSurvivedMemorySize could then also just call MonitoringSurvivedProcessMemorySize, right?

Reply:
Right